src/mac.m

```diff
@@ -1806,7 +1811,7 @@ - (void)windowDidExitFullScreen:(NSNotification*)notification
   [impl->drawView setFrameOrigin:drawPt.origin];
 
   // Dispatch new configuration
-  return dispatchCurrentChildViewConfiguration(view);
+  return dispatchCurrentChildViewConfiguration(view, false);
```

Comment:
Not so much false because "we [...] set size just before", but because this function must not change the size at all.